extensions/windows-event-log/Bookmark.cpp

```diff
@@ -222,7 +184,7 @@ bool Bookmark::getBookmarkXmlFromFile(std::wstring& bookmarkXml) {
   if (std::wstring::npos == pos) {
     logger_->log_error("No '!' in bookmarXml '%ls'", bookmarkXml.c_str());
     bookmarkXml.clear();
-    return createEmptyBookmarkXmlFile();
+	return false;
```

Comment:
This seems to be misindented.

Reply:
Thanks, since these are Windows-only, I edited them in VS when migrating them, and well... this is the result.

Reply:
If you find a simple way to disable automatic reformatting in VS, I'm interested as well. I sometimes have to recreate my windows changes on my linux editor to avoid huge diffs due to automatic reformatting.
There are a handful of options, but no obvious (to me) way to disable messing with unchanged code.